### DISCUSSION TITLE
Build flags depend on build platform

### DISCUSSION THREAD

**Discussion by taglme** [goreleaser/goreleaser#1509] [Q&A] [answered]
Hi!
For our application we should pass 'ldflags -H windowsgui'  only for Windows target platform build.
If this flag passed for other platforms, build is failed.
Is it possible to define custom ldflags depend on build platform?

**Comment by caarlos0** [comment, marked as answer]
you can remove windows from your build `goos` list and add another build section with only windows on `goos`... 

you'll need to wire the `ids` in some places, but it should work :)
